airflow/models.py

```diff
@@ -638,7 +636,7 @@ def get_password(self):
         if self._password and self.is_encrypted:
             try:
                 fernet = get_fernet()
-            except:
+            except Exception:
```

Comment:
Looking at this one. It will raise an `AirflowException`. I don't see why we don't raise the original exception:
https://github.com/feng-tao/incubator-airflow/blob/49a5b88dc99a2f987022e976be509cf1dba71d32/airflow/models.py#L104-L121